packages/vagov-eslint/.eslintignore

```diff
@@ -4,4 +4,5 @@
 **/node_modules/*
 **/.cache/
 **/*.config.*
-**/*.conf.*
\ No newline at end of file
+**/*.conf.*
+/packages/formation-react/*.js
```

Comment:
might revisit this ignore file later and require each package to maintain it's own .eslintignore